src/app/utils/functions/objectFunctions.ts

```diff
@@ -1,16 +1,14 @@
 import { compareDates, isValidDate } from './dateFunctions';
-import moment, { Moment } from 'moment';
 import { isEmpty } from './validateFunctions';
+import { Moment } from 'moment';
 
 const isObject = (object: unknown): boolean => object != null && typeof object === 'object';
```

Comment:
I know this is not part of the change, but is it a good idea to move this function to validateFunctions.ts?
Might be helpful in app use as well.